caphandler/managers.go

```diff
@@ -1,171 +0,0 @@
-	"github.com/ODIM-Project/PluginCiscoACI/capresponse"
-	"github.com/ODIM-Project/PluginCiscoACI/caputilities"
-	pluginConfig "github.com/ODIM-Project/PluginCiscoACI/config"
-	iris "github.com/kataras/iris/v12"
-)
-
-//GetManagersCollection  Fetches details of the given resource from the device
-func GetManagersCollection(ctx iris.Context) {
-	//Get token from Request
-	token := ctx.GetHeader("X-Auth-Token")
-	uri := ctx.Request().RequestURI
-	//Validating the token
-	if token != "" {
-		flag := TokenValidation(token)
-		if !flag {
-			log.Println("Invalid/Expired X-Auth-Token")
-			ctx.StatusCode(http.StatusUnauthorized)
-			ctx.WriteString("Invalid/Expired X-Auth-Token")
-			return
-		}
-	}
-	var deviceDetails capmodel.Device
-	// if any error come while getting the device then request will be for  plugins manager
-	ctx.ReadJSON(&deviceDetails)
-	if deviceDetails.Host == "" {
-		var members = []capresponse.Link{
-			capresponse.Link{
-				Oid: "/ODIM/v1/Managers/" + pluginConfig.Data.RootServiceUUID,
-			},
-		}
-
-		managers := capresponse.ManagersCollection{
-			OdataContext: "/ODIM/v1/$metadata#ManagerCollection.ManagerCollection",
-			//Etag:         "W/\"AA6D42B0\"",
-			OdataID:      uri,
-			OdataType:    "#ManagerCollection.ManagerCollection",
-			Description:  "Managers view",
-			Name:         "Managers",
-			Members:      members,
-			MembersCount: len(members),
-		}
-		ctx.StatusCode(http.StatusOK)
-		ctx.JSON(managers)
-		return
-	}
-	getInfoFromDevice(uri, deviceDetails, ctx)
-	return
-
-}
-
-//GetManagersInfo Fetches details of the given resource from the device
-func GetManagersInfo(ctx iris.Context) {
-	//Get token from Request
-	token := ctx.GetHeader("X-Auth-Token")
-	uri := ctx.Request().RequestURI
-
-	//Validating the token
-	if token != "" {
-		flag := TokenValidation(token)
-		if !flag {
-			log.Println("Invalid/Expired X-Auth-Token")
-			ctx.StatusCode(http.StatusUnauthorized)
-			ctx.WriteString("Invalid/Expired X-Auth-Token")
-			return
-		}
-	}
-	var deviceDetails capmodel.Device
-	// if any error come while getting the device then request will be for  plugins manager
-	ctx.ReadJSON(&deviceDetails)
-	if deviceDetails.Host == "" {
-		managers := capresponse.Manager{
-			OdataContext: "/ODIM/v1/$metadata#Manager.Manager",
-			//Etag:            "W/\"AA6D42B0\"",
-			OdataID:         uri,
-			OdataType:       "#Manager.v1_3_3.Manager",
-			Name:            pluginConfig.Data.PluginConf.ID,
-			ManagerType:     "Service",
-			ID:              pluginConfig.Data.RootServiceUUID,
-			UUID:            pluginConfig.Data.RootServiceUUID,
-			FirmwareVersion: pluginConfig.Data.FirmwareVersion,
-			Status: &capresponse.ManagerStatus{
-				State: "Enabled",
-			},
-		}
-		ctx.StatusCode(http.StatusOK)
-		ctx.JSON(managers)
-		return
-	}
-	getInfoFromDevice(uri, deviceDetails, ctx)
-	return
-
-}
-
-func getInfoFromDevice(uri string, deviceDetails capmodel.Device, ctx iris.Context) {
```

Comment:
from this file, only  this function needs to be removed as per the discussion, the   other function needs to retained   since  it will provide the response     plugin manager information

Reply:
Readded this in this PR. Removing the body will be taken care of in the subsequent PR.